### PR TITLE
Upgrade to NodeJS 12

### PR DESCRIPTION
**Pull request by The-smooth-operator**

### CODE REVIEW

**Comment by The-smooth-operator** [comment]
@akatsoulas let's get into this once we are done moving out from Mozillians backend

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN adduser --uid 431 --disabled-password --disabled-login --gecos 'webdev' --no
 
 RUN apt-get update && \
```
Comment (akatsoulas) [inline Dockerfile:9]: Let's also update the image from stretch to buster.

File: .travis.yml [hunk 2]
```diff
@@ -9,7 +9,9 @@ services:
 - docker
 before_install:
 - pip install flake8
-- npm install -g eslint stylelint
+- nvm install 12.18.3
```
Comment (akatsoulas) [inline .travis.yml:12]: We can also use version `12.8.1` which comes pre-installed.

File: .travis.yml [hunk 3]
```diff
@@ -9,7 +9,9 @@ services:
...
+- nvm use 12.18.3
+- npm install -g eslint
```
Comment (akatsoulas) [inline .travis.yml:14]: shall we also add back `stylelint`?

File: Dockerfile (hunk 1 above)
Comment (The-smooth-operator) [reply Dockerfile:9]: :+1:

File: .travis.yml (hunk 3 above)
Comment (The-smooth-operator) [reply .travis.yml:13]: trying

File: .travis.yml (hunk 2 above)
Comment (The-smooth-operator) [reply .travis.yml]:
```
You need to run "nvm install 12.18.1" to install it before using it.

The command "nvm use 12.18.1" failed and exited with 3 during .
```

**Pull request merged**